tst/EnergyPlus/unit/CommandLineInterface.unit.cc

```diff
@@ -102,12 +102,24 @@ class CommandLineInterfaceFixture : public EnergyPlusFixture
     {
         EnergyPlusFixture::SetUpTestCase();
 
-        // For the "legacy" mode, we need in.idf / in.epw in the current directory
-        auto inputFilePath = configured_source_directory() / "tst/EnergyPlus/unit/Resources/UnitaryHybridUnitTest_DOSA.idf";
-        fs::copy_file(inputFilePath, FileSystem::getAbsolutePath("in.idf"), fs::copy_options::skip_existing);
+        // For the "legacy" mode, we need *any* in.idf / in.epw in the current directory
+        // This is done via cmake for the CTest case at least, but if you run the energyplusapi_tests exe directly, the current directory isn't
+        // necessarily the <build>/tst/unit/EnergyPlus one, so we do it here anyways
+        {
+            auto destPath = FileSystem::getAbsolutePath("in.idf");
+            if (!fs::is_regular_file(destPath)) {
+                auto inputFilePath = configured_source_directory() / "tst/EnergyPlus/unit/Resources/UnitaryHybridUnitTest_DOSA.idf";
+                fs::copy_file(inputFilePath, destPath, fs::copy_options::skip_existing);
+            }
+        }
 
-        auto inputWeatherFilePath = configured_source_directory() / "weather/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw";
-        fs::copy_file(inputWeatherFilePath, FileSystem::getAbsolutePath("in.epw"), fs::copy_options::skip_existing);
+        {
+            auto destPath = FileSystem::getAbsolutePath("in.epw");
+            if (!fs::is_regular_file(destPath)) {
+                auto inputWeatherFilePath = configured_source_directory() / "weather/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw";
+                fs::copy_file(inputWeatherFilePath, destPath, fs::copy_options::skip_existing);
+            }
+        }
```

Comment:
Test existence before copying. This handles the case you run the `energyplusapi_tests` .exe directly, in which case the current directory is whichever directory you are currently in.

Reply:
OK, this seems sufficient.  If we are doing it via ctest, the file will already be there.  If running by the gtest binary, it likely won't be multithreaded, so this shouldn't race.  I suppose there may be a super weird case where this still causes a race, but it's going to be crazy rare.  This is fine.